vcpkg/consume/binary-caching-default.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Default local vcpkg binary cache"
+description: Learn about the default vcpkg binary cache
+author: vicroms
+ms.author: viromer
+ms.date: 9/25/2023
+ms.prod: vcpkg
+ms.topic: concept-article
+---
+# Default local vcpkg binary cache
+
+By default, vcpkg creates a local binary cache. This saves time when installing a package that has been previously installed in the same machine.
+
+The location of the default binary cache depends on your operating system.
+
+## [Windows](#tab/default-windows)
```

Comment:
Tabs or article pivots? When should we use one over the other?

Reply:
I went with tabs here since the workflow doesn't change, only the specific syntax for each step.

Reply:
It might be nicer just to add the article pivots. That way, the reader just has to select their operating system rather than having to select it multiple times throughout the articles.